arevyn24-assignment-4/src/arevyn24/assignment/pkg4/Assignment4Methods.java

```diff
@@ -0,0 +1,329 @@
+        while (input != 6) {
+            
+            printMenu();
+            print("\nEnter a menu number: ");
+            input = keyboard.nextInt();
+           
+            switch(input) {
+                case 1:
+                    wordValueCalc();
+                    break;
+                case 2:
+                    gradeCalc(keyboard);
+                    break;
+                case 3:
+                   loopingLyrics(keyboard);
+                    break;
+                case 4:
+                    slotMachine(keyboard);
+                    break;
+                case 5:
+                    reverseWord(keyboard); ;
+                    break;
+                case 6:
+                    print("EXIT\n");
+                    break;
+                default:
+                    print("Oops! Try again.\n");  
+                    break;
+            }  //end switch  
+        }   //end while
+        print("\tProgram Ended - Goodbye!\n");
+               
+    }       // end main method
+    
+    
+    /**
+     * Print method prints a string value to the console
+     * @param msg string value
+     */
+    public static void print(String msg) {
+        System.out.print(msg);
+    }   //end print method
+    
+    
+    /**
+     * PrintMenu() prints the array items to the console using a for loop
+     */
+    public static void printMenu() {
+        String[] menuItems = {
+           "Word Value Calculator", "Grade Calculator","Song Lyrics Loop",
+        "Slot Machine Simulator","Reverse Words", "Exit"
+       };
+       
+        print("\t\tMENU\n\n");
+        
+       for(int i=0; i < menuItems.length; i++) {
+           print((i+ 1) + ". " + menuItems[i]+"\n");
+       }
+    };  //end printMenu method
+    
+    
+    /**
+    * wordValueCalc() accepts a word, retrieves the numeric value of each
+    * character in the word, adds the sum of their values, and prints the sum. 
+    */
+    public static void wordValueCalc() {
+      //ASSIGNMENT 1 CODE
+        char uprH = 'H';
+        char lwrU = 'u';
+        char lwrF = 'f';
+        char lwrL = 'l';
+        char lwrE = 'e';
+        char lwrP = 'p';
+        int asciiSum;
+        
+        print("WORD CALCULATOR\n"
+                + "\tThe word is 'Hufflepuff'\n");
+        print("\t" + uprH + " " + lwrU + " " + lwrF + " " + lwrF + " " +
+                lwrL + " " + lwrE + " " + lwrP + " " + lwrU + " " + lwrF +
+                 " " + lwrF+"\n");
+
+        asciiSum = uprH + lwrU + lwrF + lwrF + lwrL + lwrE + lwrP + 
+                lwrU + lwrF + lwrF;
+
+        print("\tThe sum of the word is: " + asciiSum + "\n");
+    };      //end wordValueCalc method
+    
+    
+    /**
+     * gradeCalc() accepts a value from the user and calculates and prints their
+     * score, percentage and letter grade
+     * @param keyboard Scanner keyboard reads user input 
+     */
+    public static void gradeCalc(Scanner keyboard) {
```

Comment:
This is a fine method I like the reuses of Scanner. When we do objects this will be a pattern we will extract a lot. :) 

The feedback I will give is that this single method is rather large, like its own program really. I know that is exactly what I asked you to do but we are going to introduce the way to resolve this issue tonight.